### ISSUE TITLE
Feature: Ignore quotes of strings when comparing them

### ISSUE THREAD

**Issue by haocheng6** [azat-io/eslint-plugin-perfectionist#52]
### What rule do you want to change?

sort-array-includes

### Describe the problem

Currently the `sort-array-includes` rule doesn't ignore the surrounding quotes of string literals when comparing them. This will produce unexpected errors in some cases.

### Code example

- index.js
```js
function isValid(name) {
  return ['Burger King', "McDonald's", 'Subway'].includes(name);
}
```

- .eslintrc.json
```json
{
  "plugins": ["perfectionist"],
  "rules": {
    "perfectionist/sort-array-includes": ["error"],
    "quotes": ["error", "single", { "avoidEscape": true }]
  }
}
```

### Additional comments

Current behavior:

>   2:40  error  Expected "'Subway'" to come before ""McDonald's""  perfectionist/sort-array-includes

Expected behavior: The `sort-array-includes` rule should not complain about the code snippet above.

### Validations

- [X] Read the [docs](https://eslint-plugin-perfectionist.azat.io).
- [X] Check that there isn't [already an issue](https://github.com/azat-io/eslint-plugin-perfectionist/issues) that reports the same bug to avoid creating a duplicate.

**Comment by azat-io** [comment]
Thank you!

Fixed in ec2e2f5a989c124d51bc42ef8fd86d3b477d1b2d.

Will be released soon.

**Comment by azat-io** [comment]
Released in [v2.0.0](https://github.com/azat-io/eslint-plugin-perfectionist/releases/tag/v2.0.0)
